reconcile/utils/oc.py

```diff
@@ -1138,3 +1152,42 @@ def __init__(self, log_level, message):
 
     def __bool__(self):
         return False
+
+
+def validate_labels(labels) -> Iterable[str]:
```

Comment:
If we're going to type the output, worth typing the input as well?

Also, worthy of a unit test?  Seems like something that is simple enough to test (no external dependencies to mock).

Reply:
yes, you're right, i'll do that